dali/base/dadfs.cpp

```diff
@@ -5306,9 +5312,11 @@ class CDistributedSuperFile: public CDistributedFileBase<IDistributedSuperFile>
         init(_parent,tree,_name,user,transaction);
     }
 
-    CDistributedSuperFile(CDistributedFileDirectory *_parent, IPropertyTree *_root)
+    CDistributedSuperFile(CDistributedFileDirectory *_parent, IPropertyTree *_root, const char *optionalName)
     {
         commonInit(_parent, _root);
+        if (optionalName)
+            logicalName.set(optionalName);
```

Comment:
This constructor is public. What happens if the optionalName is an empty string?